pkg/rhcos/openstack.go

```diff
@@ -41,3 +43,39 @@ func GenerateOpenStackImageName(rhcosImage, infraID string) (imageName string, i
 
 	return infraID + "-rhcos", true
 }
+
+// ValidateOverriddenOpenStackImageName checks if an overridden image name exists and there are no other images with this name.
+func ValidateOverriddenOpenStackImageName(imageName, cloud string) error {
```

Comment:
please move this to `pkg/tfvars/openstack`, that seems like the best place for this.

Also would like to recommend this be private function and not part of the package API

Reply:
Thanks! done